app/src/utils/datetime.ts

```diff
@@ -1,3 +1,9 @@
+import dayjs from 'dayjs';
+import duration from 'dayjs/plugin/duration';
+import { pluralize } from './Utils';
+
+dayjs.extend(duration);
```

Comment:
Is this being leveraged at all?

Reply:
No idea if this is sufficient or not, but came across this plugin: https://day.js.org/docs/en/plugin/relative-time

Reply:
Oops, leftover from an earlier version